### PR TITLE
[docs] Release notes for 2.20.2.1-b3

### PR DESCRIPTION
**Pull request by aishwarya24**

### CODE REVIEW

**Review by ddhodge — COMMENTED** [review]
No ticket numbers?

**Review by ddhodge — APPROVED** [review]
LGTM if you add the version number update

File: docs/content/preview/releases/ybdb-releases/v2.20.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ For more information, see [#20827](https://github.com/yugabyte/yugabyte-db/issue
 
 {{< /warning >}}
 
-## v2.20.2.1 - March 15, 2024 {#v2.20.2.1}
+## v2.20.2.1 - March 22, 2024 {#v2.20.2.1}
 
 **Build:** `2.20.2.1-b1`
```
Comment (ddhodge) [inline docs/content/preview/releases/ybdb-releases/v2.20.md:63]: not b3?

Comment (aishwarya24) [reply docs/content/preview/releases/ybdb-releases/v2.20.md:63]: Thanks. Missed this.

**Review by ddhodge — APPROVED** [review]

**Pull request merged**